sympy/core/function.py

```diff
@@ -2071,9 +2087,21 @@ class Subs(Expr):
     """
     Represents unevaluated substitutions of an expression.
 
-    ``Subs(expr, x, x0)`` receives 3 arguments: an expression, a variable or
-    list of distinct variables and a point or list of evaluation points
-    corresponding to those variables.
+    ``Subs(expr, x, x0)`` represents the expression resulting
+    from substituting x with x0 in expr.
+
+    Parameters
+    ==========
+
+    expr : Expr
+        An expression.
+
+    x : tuple, variable
+        A variable or list of distinct variables.
+
+    x0 : tuple or list of tuples
+        A point or list of evaluation points
+        corresponding to those variables.
 
     ``Subs`` objects are generally useful to represent unevaluated derivatives
```

Comment:
![Screen Shot 2020-10-01 at 18 46 11](https://user-images.githubusercontent.com/34944973/94794108-6548e600-0416-11eb-800c-0064aa2170af.png)
This needs to be fixed by adding new sections (maybe examples or notes) below because what you'd really get would look like that.
I'd note that `Parameters` section needs more correct scoping, or it can cause breaks like that